strax/processing/peak_splitting.py

```diff
@@ -26,14 +26,10 @@ def split_peaks(peaks, records, to_pe, algorithm='local_minimum',
     splitter = dict(local_minimum=LocalMinimumSplitter,
                     natural_breaks=NaturalBreaksSplitter)[algorithm]()
 
-    if data_type == 'hitlets':
-        # This is only needed once.
-        _, next_ri = strax.record_links(records)
-    elif data_type == 'peaks':
-        next_ri = None
-    else:
+    data_type_is_not_supported = data_type not in ('hitlets', 'peaks')
+    if data_type_is_not_supported:
         raise TypeError(f'Data_type "{data_type}" is not supported.')
```

Comment:
I do appreciate your pilgrimage for clear code, but perhaps this is not really necessary right 😉 
```suggestion
    if data_type not in ('hitlets', 'peaks'):
        raise TypeError(f'Data_type "{data_type}" is not supported.')
```